src/main/java/javaserver/RequestParser.java

```diff
@@ -19,6 +20,21 @@ public static String getRequestURI(String rawRequest) {
 			return "";
 		}
 	}
+	
+	public static String getRequestData(String rawRequest) {
+		String data = "";
+		try {
+			String[] requestLines = splitRequest(rawRequest);
+			for (int i = 2; i < requestLines.length; i++) {
+				if (requestLines[i].contains("=")) {
+					data += requestLines[i] + "\n";
+				}
+			}
+			return data;
+		} catch (ArrayIndexOutOfBoundsException e) {
+		}
+		return data;
```

Comment:
You can delete the return on 33 since you are doing it here at the end of the method